configs/mmdet/detection/detection_ipu_static-800x1344.py

```diff
@@ -0,0 +1,8 @@
+_base_ = ['../_base_/base_static.py']
+
+backend_config = dict(type='ipu', precision='fp16', output_dir='',
+                      batches_per_step=1, ipu_version='ipu21', input_shape='input=1,3,320,320', eightbitsio='',
+                      popart_options=dict(rearrangeAnchorsOnHost='false', enablePrefetchDatastreams='false', groupHostSync='false', partialsTypeMatMuls="half"))
```

Comment:
Since many of the settings are the same as the `base_static.py`, there is no need to write them again.

Reply:
resolved